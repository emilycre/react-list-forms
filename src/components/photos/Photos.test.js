import React from 'react';
import { shallow } from 'enzyme';
import Photos from './Photos';

describe('Photos component', () => {
  it('Renders Photos', () => {
    const photos = [
      '/img1',
      '/img2',
      '/img3'
    ];

    const wrapper = shallow(<Photos photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
