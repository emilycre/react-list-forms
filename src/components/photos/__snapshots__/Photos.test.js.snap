// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photos component Renders a Photos 1`] = `
<ul>
  <li
    key="/img1"
  >
    <Photo
      photo="/img1"
    />
  </li>
  <li
    key="/img2"
  >
    <Photo
      photo="/img2"
    />
  </li>
  <li
    key="/img3"
  >
    <Photo
      photo="/img3"
    />
  </li>
</ul>
`;
